libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -421,6 +421,11 @@ class AP_AHRS {
         return _ekf_type;
     }
 
+    // set the selected ekf type, for RC aux control
+    void set_ekf_type(uint8_t ahrs_type) {
+        _ekf_type.set(ahrs_type);
+    }
```

Comment:
Using the enum (and const!) would protect against accidentally setting a bogus EKF type. 

```suggestion
    // set the selected ekf type, for RC aux control
    void set_ekf_type(const EKFType ahrs_type) {
        _ekf_type.set((uint8_t)ahrs_type);
    }
```

Reply:
changed